### PR TITLE
fix: use env_var_name instead of string to secure sensitive data

### PR DESCRIPTION
**Pull request by brunohkbx**
CircleCI [best practices](https://circleci.com/docs/2.0/orbs-best-practices/#parameters) suggests to use `env_var_name` to secure  sensitive data.

### CODE REVIEW

**Comment by sue445** [comment]
@brunohkbx This is breaking changes.

When I set plain user name to `git_user_name`, build is error. https://circleci.com/gh/sue445/circleci-ruby-orbs-test/1042

# Example
```yaml
    jobs:
      - ruby-orbs/bundle-update-pr:
          git_user_name:  "CircleCI bundle updater"
```

`github_access_token ` and `enterprise_octokit_access_token ` are sensitive data, but other not. (e.g. `git_user_name `)

I think sensitive data should store to secure variables, but I want to write non-sensitive data directly in yaml.

**Comment by brunohkbx** [comment]
@sue445 you are right. Could you please review it again?
I'm just unsure about `enterprise_octokit_api_endpoint`. Do you think it's ok to expose it in the yaml?

**Comment by sue445** [comment]
> I'm just unsure about enterprise_octokit_api_endpoint. Do you think it's ok to expose it in the yaml?

I think this is no problem. When use `circleci-bundle-update-pr` with CircleCI + GH:E, code is hosted in protected GH:E

File: src/commands/run-circleci-bundle-update-pr.yml [hunk 1]
```diff
@@ -31,14 +31,14 @@ parameters:
     default: "$CIRCLE_BRANCH"
     description: "Space separated branches. (e.g. 'master develop topic')"
   github_access_token:
-    type: string
-    default: "$GITHUB_ACCESS_TOKEN"
+    type: env_var_name
+    default: GITHUB_ACCESS_TOKEN
     description: |
       Your GitHub personal access token.
```
Comment (sue445) [inline src/commands/run-circleci-bundle-update-pr.yml]: Please update description and usage in https://github.com/sue445/circleci-ruby-orbs/blob/1.4.3/README.md#parameters-1.  

e.g. `Your GitHub personal access token key`, `Your GitHub personal access token env name` or any other good expression.

**Comment by brunohkbx** [comment]
Done @sue445

**Comment by sue445** [comment]
LGTM!

**Pull request merged**